challenges/fullstack/challenge.md

```diff
@@ -19,148 +14,124 @@ We would like to evaluate your skill in the following areas:
 We believe this technique is not only better but also more fun compared to
 whiteboard/quiz interviews so common in the industry. It’s not without the
 downsides - it could take longer than traditional interviews. That said, it's
-our view that this type of challenge gives us a more accurate assessment of
-your ability to work well on the types of projects we’re working on day-to-day
-here at Teleport. [Some of the best teams use coding
-challenges](https://sockpuppet.org/blog/2015/03/06/the-hiring-post/).  We
+our view that this type of challenge gives us a more accurate assessment of your
+ability to work well on the types of projects we’re working on day-to-day here
+at Teleport. [Some of the best teams use coding
+challenges](https://sockpuppet.org/blog/2015/03/06/the-hiring-post/). We
 appreciate your time and are looking forward to hacking on this project
 together.
 
+## Summary
+
+In this challenge, you will build an application that allows a user to browse
+directory content on a remote server. Think Google Drive, Dropbox, or even
+GitHub's file browser.
+
+<img src="./assets/google.jpg" height="300" />
+<img src="./assets/github.jpg" height="200" />
+
 ## Tools
 
-* CSS: any
-* Frontend: React, React Router
-* Backend: NodeJS, Go, Rust
-* Version Control: GitHub
+In order to provide an experience similar to working on Teleport itself, we've
+selected tools that align with our internal development environment.
+
+* Version control and code review performed via GitHub
+* The backend API is written in Go
+* The frontend is a React app written in TypeScript
+
+At Teleport, we use [styled components](https://styled-components.com) for
+styling, but you are not required to use them for this challenge.
 
 ## Levels
 
-There are 6 engineering levels at Teleport. Levels 1-4 are open through the
-fullstack challenge.
+There are 6 engineering levels at Teleport. Levels 1-4 are open to external candidates.
 
 For fullstack engineers, levels 5 and 6 are only for internal promotions. Check
 [Fullstack Engineering Levels](../../levels/fullstack.pdf) for more details.
 
-## Requirements
+At this time, we are seekling level 4 full stack engineers only.
+
+## Requirements - Level 4
```

Comment:
Should we call out the security requirements in this block? I know they're mentioned later, but mitigating against common web security issues feels like it should be highlighted.